content/cloud/deployment-concepts.md

```diff
@@ -8,54 +8,46 @@ url = "https://github.com/fermyon/developer/blob/main/content/cloud/deployment-c
-  - Once a bindle is put into a bindle hub (the server), it cannot be changed, nor can it be deleted.
-- Bindles can use semantic versioning [SemVer](https://semver.org) as part of their names.
-
-## The Deployment Process Explained
-
-In the Fermyon Cloud, we host an instance of Bindle, so when you run `spin deploy`, the command will take care of:
-1. Packaging the application and upload it to the Fermyon Cloud
-2. Creating or upgrade an application, using the bindle
-
-There is no direct interaction with Bindle when using the Fermyon Cloud.
+It's important to note that Spin applications deployed using the Cloud Plugin, are not, however, compatible with being run by an [OCI runtime](https://github.com/opencontainers/runtime-spec) compatible runtime, like [runc](https://github.com/opencontainers/runc). If you want to explore running Spin applications using a container runtime, you can do so using [`containerd` and `runwasi`](https://github.com/containerd/runwasi).
 
-Let's unfold each of these steps.
+> Note: The Fermyon Cloud Spin plugin supports upgrading an application which was deployed using an version older than 0.3.0. However, you cannot use a version prior to 0.3.0 to upgrade an application, which was already deployed using 0.3.0.
 
-### 1. Packaging and Uploading an Application
-
-The first step in deploying an application is to package all the files into parcels and generate an invoice.  All of this is handled in a staging directory, which is either a temporary directory using [this implementation](https://doc.rust-lang.org/std/env/fn.temp_dir.html#platform-specific-behavior) or the staging directory defined using the `--staging-dir` option.
-
-The bindle will be named using the `name` from `spin.toml`, the `version` from `spin.toml`, and a build metadata string, which is automatically generated by Spin at deployment time.
-
-Bindles in the Fermyon Cloud always use Semantic Versioning and require major, minor, and patch version numbers to be specified. Therefore, the version in `spin.toml` has to conform to the `MAJOR.MINOR.PATCH` format, i.e. `1.0.1` is valid, `1.0` and `1` are not valid version numbers for a Spin application. The result is that the version of a Spin application will be like this `my_app_name/1.0.0+r80e5abb`.
+## The Deployment Process Explained
 
-Following packaging, the bindle will be uploaded to the Fermyon Cloud. As soon as a bindle has been uploaded, it cannot be modified or deleted. This is to preserve the integrity of the immutability of bindles.
+When you run the `spin cloud deploy` command, the following happens:
 
-### 2. Create or Upgrade an Application
+1. Packaging the application and uploading it to the Fermyon Cloud
+  - The first step in deploying an application is to package all the files using the OCI specification. The Cloud Plugin will create individual layers for the files that make up the application, to optimize changes to applications on subsequent packaging and deployment.
```

Comment:
The latter sentence here may not continue to be strictly accurate in the future.  (Individual files may not correspond 1:1 with layers; i.e. we may bundle multiple into a given layer.)  We can leave this in if we'd like and update when the implementation changes -- or we might consider rewording.